bokeh/models/ui/__init__.py

```diff
@@ -26,12 +25,14 @@
     inspector,
     menus,
     tooltips,
+    ui_element,
```

Comment:
I needed to add this import to have the `UIElement` class included in the ref guide as well